src/functions-and-arrays.js

```diff
@@ -52,16 +99,46 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(array) {
+  if (array.length === 0) {
+    return null;
+  } else {
+    let newArr = [];
+    for (let i = 0; i < array.length; i++) {
+      let currentWord = array[i];
+      console.log(currentWord);
+      if (newArr.indexOf(currentWord) < 0) {
+        newArr.push(currentWord);
+      }
+    }
+    return newArr;
+  }
+}
+
+console.log(uniquifyArray(wordsUnique))
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
-
+function doesWordExist(arr, searchWord) {
+  if (arr.length === 0) {
+    return null;
+  } else {
+    let match = false;
+
+    for (let i = 0; (i < arr.length) & !match; i++) {
+      if (arr[i] === searchWord) {
+        match = true;
```

Comment:
You could also move the return-statement into the loop (in line 133), then you don't need the `match`-variable and the `break`.